floop-data-import.py

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
Not sure why this existing file from Sprint1 is included as a new file as part of this PR?